include/seqan3/argument_parser/detail/version_check.hpp

```diff
@@ -403,7 +403,7 @@ class version_checker
 )";
             return true; // default: check version if you cannot ask the user
         }
-    }
+    } // LCOV_EXCL_STOP
```

Comment:
You place it here, because the current line is not part of the excluded section. However, it is a slightly confusing layout because the corresponding bracket for the excluded if/else block is one line above. My idea: Insert the comment in its own line between the closing brackets and use the indentation of the if/else block.